xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -549,13 +552,12 @@ COverlay* CRenderer::ConvertLibass(
   overlay = new COverlayQuadsDX(images, rOpts.frameWidth, rOpts.frameHeight);
 #endif
 
-  // scale to video dimensions
   if (overlay)
   {
-    overlay->m_width = rOpts.frameWidth / rOpts.videoWidth;
-    overlay->m_height = rOpts.frameHeight / rOpts.videoHeight;
-    overlay->m_x = (rOpts.videoWidth - rOpts.frameWidth) / 2 / rOpts.videoWidth;
-    overlay->m_y = (rOpts.videoHeight - rOpts.frameHeight) / 2 / rOpts.videoHeight;
+    overlay->m_width = 1.0;
```

Comment:
Are these sane defaults? If so I think they could be moved to the constructor of the overlay, both are only used in this class